parsers/CA_SK.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import Optional
+
+# The arrow library is used to handle datetimes consistently with other parsers
+import arrow
```

Comment:
```suggestion
from pytz import utc
```
Since we only get the current datetime let's get rid of arrow and just use datetime + pytz.